src/pineko/check.py

```diff
@@ -131,16 +153,16 @@ def is_fonll_b(fns, lumi):
     bool
         true if the fktable is a FONLL-B DIS fktable
     """
-    for lists in lumi:
-        for el in lists:
-            if (not islepton(el[0])) and (not islepton(el[1])):
-                # in this case we are sure it is not DIS so for sure it is not FONLL-B
-                return False
-    if fns == "FONLL-B":
+    if is_dis(lumi) and fns == "FONLL-B":
         return True
     return False
 
 
+def issimFONLL(FNS):
```

Comment:
there is still lot's of `simFONLL` around (Firefox counts 18 occurences) - while changing, please make `snake_case`